hydra-plutus/src/Plutus/Orphans.hs

```diff
@@ -47,3 +54,24 @@ instance FromJSON POSIXTime where
 
 instance Arbitrary a => Arbitrary (UpperBound a) where
   arbitrary = upperBound <$> arbitrary
+
+instance ToJSON Plutus.PubKeyHash where
+  toJSON = \kh ->
+    object
+      [ "tag" .= Aeson.String "PubKeyHash"
+      , "keyHash" .= Aeson.String (decodeUtf8 $ Base16.encode $ fromBuiltin $ getPubKeyHash kh)
+      ]
+
+instance FromJSON Plutus.PubKeyHash where
+  parseJSON = withObject "PubKeyHash" $ \o -> do
+    tag <- o .: "tag"
+    case tag :: Text of
+      "PubKeyHash" -> do
+        hexText :: Text <- o .: "keyHash"
+        case Base16.decode $ encodeUtf8 hexText of
+          Left e -> fail e
+          Right bs -> pure $ Plutus.PubKeyHash (toBuiltin bs)
+      _ -> fail "Expected tag to be PubKeyHash"
```

Comment:
Should: add roundtrip tests for these instances.

Reply:
is it not included as part of [this roundtrip test](https://github.com/input-output-hk/hydra-poc/blob/46aaf61202eba44f349db3174aeca7f0b99b134a/hydra-node/test/Hydra/Chain/Direct/StateSpec.hs#L115)?

Reply:
do u want to explicitly add a roundtrip for this?
```hs
describe "Plutus.PubKeyHash" $
    roundtripAndGoldenSpecs (Proxy @Plutus.PubKeyHash)
```

Reply:
Can't hurt to add it, but yes. You are right that it should be part of the chainstate roundtrip.

Reply:
Ok will add it.